client/app/scripts/utils/router-utils.js

```diff
@@ -90,21 +90,20 @@ export function updateRoute(getState) {
 
 
 export function getRouter(dispatch, initialState) {
+  let mergedState = initialState;
   // strip any trailing '/'s.
   page.base(window.location.pathname.replace(/\/$/, ''));
 
+  const storageState = storageGet(STORAGE_STATE_KEY);
+  if (storageState) {
+    window.location.hash = `!/state/${storageState}`;
+    const parsedState = JSON.parse(decodeURL(storageState));
+    mergedState = Object.assign(initialState, parsedState);
```

Comment:
Also worth noting that Scope didn't have the `transform-object-rest-spread` babel plugin. Also worth noting that the service-ui eslint loader looks at the `.eslint` file of the Scope repo instead of its own (probably a bug).